src/pages/Profile/Profile.jsx

```diff
@@ -1,19 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-// import person from '../../images/person.png';
+import person from '../../images/person.png';
 // ^^ blank avatar for profile pic, uncomment the above line to use
 import './Profile.css';
 
 const Profile = ({ firstName, lastName, stats }) => {
+  const displayStats = stats.map(s => {
+    return (
```

Comment:
If your arrow function has a single return statement (and nothing else in the body), you can get rid of the `return` keyword as such:
`s => { return <div ...>...</div> }`
to
`s => (<div ...>...</div>)`